examples/PhysicsLabFirmware/PhysicsLabFirmware.ino

```diff
@@ -234,6 +261,15 @@ void updateSubscribedCharacteristics() {
 #endif
     resistanceCharacteristic.writeValue(resistanceAvg);
   }
+
+  if (batteryLevel.subscribed()) {
+    float iovcc = getIoVcc();
```

Comment:
I'd suggest one minor change here to send the battery level every N milliseconds instead of every loop.

Reply:
Ignore my previous comment, `updateSubscribedCharacteristics()` is wrapped in this behaviour already :)